crates/sequencer/src/sequencer.rs

```diff
@@ -622,11 +622,29 @@ where
     pub async fn resubmit_pending_commitments(&mut self) -> anyhow::Result<()> {
         let pending_db_commitments = self.ledger_db.get_pending_commitments_l2_range()?;
         debug!("Pending db commitments: {:?}", pending_db_commitments);
-        let pending_da_commitments = self.get_pending_da_commitments().await;
-        debug!("Pending da commitments: {:?}", pending_da_commitments);
+
+        let pending_mempool_commitments = self.get_pending_mempool_commitments().await;
+        debug!(
+            "Pending mempool commitments: {:?}",
+            pending_mempool_commitments
+        );
+
+        let last_commitment_l1_height = self
+            .ledger_db
+            .get_last_sequencer_commitment_l1_height()?
+            .unwrap_or(SlotNumber(1));
+        let mined_commitments = self
+            .get_mined_commitments_from(last_commitment_l1_height)
+            .await?;
+        debug!("Unsaved mined commitments: {:?}", mined_commitments);
+
+        let mut pending_commitments_to_remove = vec![];
+        pending_commitments_to_remove.extend(pending_mempool_commitments);
+        pending_commitments_to_remove.extend(mined_commitments);
```

Comment:
why are you adding mined_commitments  here? shouldn't they be handled seperately?

Reply:
what separate functionality is needed for mined commitments? we normally save them to pg but we decided to not deal with that since we plan to remove it anyways.

Reply:
misunderstood